### ISSUE TITLE
ShimmerDirection not changing between different Widgets

### ISSUE THREAD

**Issue by Ahmadre** [hnvn/flutter_shimmer#33]
## Demonstration (GIF)

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/18512224/91230237-71f96080-e72b-11ea-9750-85246006f54d.gif)

## Code-Snippet

```dart
if (summonDirection == SummonDirection.forward)
  Positioned.fill(
    child: AnimatedOpacity(
      duration: const Duration(milliseconds: 500),
      opacity: summoning ? 1.0 : 0.0,
      child: Container(
        child: Shimmer.fromColors(
          direction: ShimmerDirection.ttb,
          baseColor: brightness == Brightness.light ? Colors.black54 : Colors.white54,
          highlightColor: Colors.white,
          child: Image.asset(
            'assets/images/summon_arrows.png',
          ),
        ),
      ),
    ),
  ),
if (summonDirection == SummonDirection.backward)
  Positioned.fill(
    child: AnimatedOpacity(
      duration: const Duration(milliseconds: 500),
      opacity: summoning ? 1.0 : 0.0,
      child: Container(
        child: Shimmer.fromColors(
          direction: ShimmerDirection.btt,
          baseColor: brightness == Brightness.light ? Colors.black54 : Colors.white54,
          highlightColor: Colors.white,
          child: RotatedBox(
            quarterTurns: 2,
            child: Image.asset(
              'assets/images/summon_arrows.png',
            ),
          ),
        ),
      ),
    ),
  ),
```

## Actual behavior

When I set `direction: ShimmerDirection.btt,` and use another direction in another widget like: `direction: ShimmerDirection.ttb,`, only the first direction is applied.

Every other Shimmer widget in my `StatefulWidget` is ignored.

## Expected behavior

direction is working for all Shimmers seperatly.

**Comment by hnvn** [comment]
I will have a look at it. Thanks for your report

**Comment by Ahmadre** [comment]
@hnvn thanks u rock!

**Comment by Ahmadre** [comment]
@hnvn any news?

**Comment by Ahmadre** [comment]
@hnvn it's been 1 month. I would appreciate it, if we can get any answer or something. thx

**Comment by hnvn** [comment]
ah, I'm so sorry, I am busy with my project this time. I will try to work on it this weekend.

**Comment by hnvn** [comment]
fixed in v1.1.2
